include/deal.II/distributed/grid_tools.h

```diff
@@ -130,6 +131,19 @@ namespace parallel
     exchange_cell_data_to_ghosts (const MeshType &mesh,
                                   const std::function<boost::optional<DataType> (const typename MeshType::active_cell_iterator &)> &pack,
                                   const std::function<void (const typename MeshType::active_cell_iterator &, const DataType &)> &unpack);
+
+    /**
+     * Exchange with all processors of the Mpi Comm @p mpi_communicator the vector of bounding
+     * boxes @p local_bboxes.
+     *
+     * This function was born to exchange bounding boxes describing the locally owned
+     * cells in a distributed triangulation obtained with the function
+     * GridTools::compute_mesh_predicate_bounding_box
```

Comment:
from the description i don't understand what it does. What do you mean by exchange? Is it govering all local boxes into a  single vector? You should probably clarify what is actually returned.

Reply:
Added a description, I hope it is good enough!